package.json

```diff
@@ -9,7 +9,7 @@
   },
   "scripts": {
     "install": "node bin/appsec-reinstall.js",
-    "rebuild": "node-gyp rebuild",
+    "rebuild": "node ./node_modules/node-gyp/bin/node-gyp.js rebuild",
```

Comment:
Why is this change necessary?

Reply:
As I understand it, npm ships its own (very old) version of `node-gyp` which is used instead of the locally installed one. See https://github.com/nodejs/node-gyp/blob/master/docs/Updating-npm-bundled-node-gyp.md
I haven't checked for sure where the default `node-gyp` binary is from, but what I know is that it runs a very old version when not adding this `./node_modules/...`.
For example in the other (broken) PR: https://github.com/DataDog/dd-native-appsec-js/runs/5499954004?check_suite_focus=true
![image](https://user-images.githubusercontent.com/70517128/158138075-05d0b8e1-1cc8-4592-bb57-9cfabce2fb6a.png)
(we want v9)